apps/web/app/hooks/auth/useAuthenticationPasscode.ts

```diff
@@ -98,13 +100,17 @@ export function useAuthenticationPasscode() {
 					}
 				}
 
-				if (res.data?.status !== 200 && res.data?.status !== 201) {
-					setErrors({ code: t('pages.auth.INVALID_INVITE_CODE_MESSAGE') });
-				}
+				// if (res.data?.status !== 200 && res.data?.status !== 201) {
```

Comment:
@paradoxe35 why this commented out now?

Reply:
Let me uncomment it
even though the code is no longer accurate.

Reply:
Before the server could return 200 response even if it's a bad request. I've fixed